AxonDeepSeg/apply_model.py

```diff
@@ -185,7 +186,7 @@ def apply_convnet(path_acquisitions, acquisitions_resolutions, path_model_folder
 
 def axon_segmentation(path_acquisitions_folders, acquisitions_filenames, path_model_folder, config_dict,
                       ckpt_name='model',
-                      segmentations_filenames=['AxonDeepSeg.png'], inference_batch_size=1,
+                      segmentations_filenames=[axonmyelin_suffix.name], inference_batch_size=1,
```

Comment:
Hmm, I'm not sure if using `.name` is the best strategy to get the names. I'd maybe go the `str(axonmelin_suffix)` route if you're doing this to convert the object from Path() to a str(), your intensions are clearer this way. @mariehbourget do you agree? This would go for every call of these throughout this PR.

Reply:
In some places (probably in some tests -- if I remember correctly), the `.name` property of the Path object was used to access the name, so I decided to keep that convention. But I concur with you that `str(path object)` is more readable. I can make changes, but the changes should be made in the entire codebase. 

**PS**: I will update the links of the snippets where `.name` property is used.

Reply:
If there is an agreement for using `str()` instead of `.name` shall I fix it in the entire codebase or for only my modified files?

Reply:
Fixed in commit 02fbcb7. Used `str()` instead of .name only for this PR.